paymentsource/client.go

```diff
@@ -150,16 +150,20 @@ func Verify(id string, params *stripe.SourceVerifyParams) (*stripe.PaymentSource
 
 func (s Client) Verify(id string, params *stripe.SourceVerifyParams) (*stripe.PaymentSource, error) {
 	body := &stripe.RequestValues{}
-	body.Add("amounts[]", strconv.Itoa(int(params.Amounts[0])))
-	body.Add("amounts[]", strconv.Itoa(int(params.Amounts[1])))
 
 	source := &stripe.PaymentSource{}
 	var err error
 
 	if len(params.Customer) > 0 {
+		body.Add("amounts[]", strconv.Itoa(int(params.Amounts[0])))
+		body.Add("amounts[]", strconv.Itoa(int(params.Amounts[1])))
+
 		err = s.B.Call("POST", fmt.Sprintf("/customers/%v/sources/%v/verify", params.Customer, id), s.Key, body, &params.Params, source)
 	} else {
-		err = errors.New("Only customer bank accounts can be verified in this manner.")
+		body.Add("values[]", strconv.Itoa(int(params.Amounts[0])))
```

Comment:
> Unclear to me if we should rename the old one, or rename the new one, but these are two different constructs in the API. Hope that helps?

Yep, that clears things up. Thanks!

@ob-stripe I'd probably suggest introducing a new struct like the one Stan has above and renaming the old one. The breaking change sucks, but looking at Splunk, it doesn't seem to be too widely used from the Go language right now (< 10 calls in the last week).